src/extension.ts

```diff
@@ -503,13 +492,11 @@ async function setupFeatureFlags(): Promise<void> {
 }
 
 /** Return view provider + name fragment pairs for auto-registering refresh() commands. */
-export function getRefreshableViewProviders(
-  resourcesViewProviderInstance: ResourceViewProvider | NewResourceViewProvider,
-): RefreshableTreeViewProvider[] {
+export function getRefreshableViewProviders(): RefreshableTreeViewProvider[] {
   // When adding a new refreshable view, also update the test block
   // "Refreshable views tests" in extension.test.ts.
   const refreshables = [
-    resourcesViewProviderInstance,
+    ResourceViewProvider.getInstance(),
```

Comment:
This shouldn't be here since we don't expose a top-level refresh and `src/commands/resources.ts` has the connection-row-refresh commands being registered, so we'll have to remove `confluent.resources.refresh` (but not `confluent.resources.refreshConnection`) and this line, but can be done in a small follow-up branch.